### PR TITLE
Expose PermitTunnel from sshd_config

### PR DESCRIPTION
**Pull request by andrei693**
This will expose PermitTunnel from sshd_config via ssh:sshd_config_permittunnel. Defaults to no.

### CODE REVIEW

**Comment by andrei693** [comment]
Are there any issues that still need to be resolved to have this merged?

File: README.md [hunk 1]
```diff
@@ -435,6 +435,18 @@ network goes down or the client host crashes.  This avoids infinitely hanging se
 
 - *Default*: 'yes'
 
+sshd_config_permittunnel
+-----------------------
+PermitTunnel in sshd_config.
+Specifies whether tun(4) device forwarding is allowed.  The argument must be
+``yes'', ``point-to-point'' (layer 3), ``ethernet'' (layer 2), or ``no''.
```
Comment (ghoneycutt) [inline README.md]: use single quotes around a string. You have other characters there.

**Comment by ghoneycutt** [comment]
Just a minor change to the README formatting and this is all good to merge.

File: README.md (hunk 1 above)
Comment (andrei693) [inline README.md]: Done.

**Comment by ghoneycutt** [comment]
Thank you! Released in v3.49.0

**Pull request merged**